module.json

```diff
@@ -10,7 +10,7 @@
       "name": "Frstrm"
     }
   ],
-  "version": "1.2.0",
+  "version": "1.2.1",
```

Comment:
looks like 1.2.1 is ghosted. So, new work needs to be based on 'dev' not 'main' to prevent this foolery in the future.

Reply:
bumping to 1.2.3